widgets/opal/switchbutton/org.eclipse.nebula.widgets.opal.switchbutton/src/org/eclipse/nebula/widgets/opal/switchbutton/SwitchButton.java

```diff
@@ -161,38 +155,33 @@
 	public SwitchButton(final Composite parent, final int style) {
 		super(parent, style | SWT.DOUBLE_BUFFERED);
 
-		selection = false;
+		selection = true;
 		text = "";
 		textForSelect = "On";
 		textForUnselect = "Off";
 		round = true;
 		borderColor = null;
 		focusColor = getDisplay().getSystemColor(SWT.COLOR_YELLOW);

```

Comment:
I found this yellow really... disturbing... is this by design? Maybe one of the COLOR_WIDGET_* constants would be more suitable?

Reply:
When I worked on this PR, I noticed this is weird. The next commit will fix it